control_panel_api/views.py

```diff
@@ -36,6 +37,11 @@ class UserViewSet(viewsets.ModelViewSet):
     filter_backends = (UserFilter,)
     permission_classes = (UserPermissions,)
 
+    def get_object(self):
```

Comment:
What is this for? Is it because otherwise a normal user can't see himself?

I think that's because of the `filters.UserFilter`, that's currently only filter out everything for non-superuser. If this is filtering logic, probably should be moved in `filters.UserFilter`.

Reply:
Probably `UserFilter` can now be something like:

```python
    def filter_queryset(self, request, queryset, view):
        queryset = super().filter_queryset(request, queryset, view)

        if is_superuser(request.user):
            return queryset
        elif request.user:
            return queryset.get(pk=request.user.pk)
        else:
           return queryset.none()
``` 

or something like that! (you got the idea)